partner-management-default.properties

```diff
@@ -282,15 +282,15 @@ mosip.role.pms.getpartnerscredentialtypepolicies=PARTNER,CREDENTIAL_PARTNER,PART
 mosip.role.pms.postpartnerscontactadd=PARTNER,AUTH_PARTNER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,ABIS_PARTNER,MANUAL_ADJUDICATION,MISP_PARTNER
 mosip.role.pms.putpartners=PARTNER,AUTH_PARTNER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,ABIS_PARTNER,MANUAL_ADJUDICATION,MISP_PARTNER
 mosip.role.pms.putpartnersnew=PARTNER,AUTH_PARTNER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,ABIS_PARTNER,MANUAL_ADJUDICATION,MISP_PARTNER
-mosip.role.pms.getpartnerspartnerid=PARTNER,AUTH_PARTNER,CREDENTIAL_PARTNER,RESIDENT,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,ABIS_PARTNER,SDK_PARTNER,MANUAL_ADJUDICATION,MISP_PARTNER
+mosip.role.pms.getpartnerspartnerid=PARTNER,AUTH_PARTNER,CREDENTIAL_PARTNER,RESIDENT,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,ABIS_PARTNER,SDK_PARTNER,MANUAL_ADJUDICATION,MISP_PARTNER,RESIDENT
 mosip.role.pms.getpartnersapikeyrequest=PARTNER,AUTH_PARTNER,ABIS_PARTNER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER
 mosip.role.pms.postpartnerscacertificateupload=PARTNERMANAGER,PARTNER_ADMIN
 mosip.role.pms.postpartnerscertificateupload=PARTNER,PMS_USER,AUTH_PARTNER,ABIS_PARTNER,SDK_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,CREDENTIAL_PARTNER,CREDENTIAL_ISSUANCE,ID_AUTHENTICATION,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER
 mosip.role.pms.getpartnerscertificate=PARTNER,PMS_USER,AUTH_PARTNER,ABIS_PARTNER,SDK_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,CREDENTIAL_PARTNER,CREDENTIAL_ISSUANCE,CREATE_SHARE,ID_AUTHENTICATION,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER
 mosip.role.pms.postpartnerssearch=PARTNER,PMS_USER,AUTH_PARTNER,ABIS_PARTNER,SDK_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,PARTNER_ADMIN,CREDENTIAL_PARTNER,ONLINE_VERIFICATION_PARTNER
 mosip.role.pms.postpartnerspartnertypesearch=PARTNER,PMS_USER,AUTH_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,PARTNER_ADMIN,CREDENTIAL_PARTNER,ONLINE_VERIFICATION_PARTNER
-mosip.role.pms.postpartnersfiltervalues=PARTNER,PMS_USER,AUTH_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER
-mosip.role.pms.postpartnersapikeyrequestfiltervalues=PARTNER,PARTNER_ADMIN,AUTH_PARTNER,CREDENTIAL_PARTNER,ONLINE_VERIFICATION_PARTNER
+mosip.role.pms.postpartnersfiltervalues=PARTNER,PMS_USER,AUTH_PARTNER,DEVICE_PROVIDER,FTM_PROVIDER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER,ABIS_PARTNER
+mosip.role.pms.postpartnersapikeyrequestfiltervalues=PARTNER,PARTNER_ADMIN,AUTH_PARTNER,CREDENTIAL_PARTNER,ONLINE_VERIFICATION_PARTNER,ABIS_PARTNER
```

Comment:
ABIS_PARTNER is a new role ? Is the keycloak init handling taken care ?
I see keycloak is marked as no, is keycloak getting released ?

Reply:
@dhanendra06 Please check this as well

Reply:
ABIS_PARTNER roles are present in keycloak last released version